charts/incubator/misskey/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
   repository: tccr.io/truecharts/misskey
   pullPolicy: IfNotPresent
-  tag: v13.14.2@sha256:025214f6c145d21ef267045f8f9588242d9799ac73102ac8151213691886f858
```

Comment:
That looks like a major version change.
Maybe it needs to be marked as such in the chart version too?

Reply:
I believe they have literally changed the whole versioning to a semi-date based one from what I can assume
So I have no idea if that should count at all towards bumping the chart's major version